widgetssdk/src/main/java/com/glia/widgets/view/Dialogs.java

```diff
@@ -18,12 +18,38 @@
 import com.glia.widgets.R;
 import com.glia.widgets.UiTheme;
 import com.glia.widgets.helper.Utils;
+import com.glia.widgets.view.button.BaseConfigurableButton;
 import com.glia.widgets.view.button.GliaNegativeButton;
 import com.glia.widgets.view.button.GliaPositiveButton;
 import com.google.android.material.dialog.MaterialAlertDialogBuilder;
 
 public class Dialogs {
 
+    public static AlertDialog showOptionsDialog(
+            Context context,
+            UiTheme theme,
+            String title,
+            String message,
+            String positiveButtonText,
+            String negativeButtonText,
+            View.OnClickListener positiveButtonClickListener,
+            View.OnClickListener negativeButtonClickListener,
+            DialogInterface.OnCancelListener cancelListener
+    ) {
+        return showOptionsDialog(
```

Comment:
I would suggest adding a new dialog for "End engagement" instead of customizing and reusing the existing one.

My points for such a solution for this case are:
- no variables like `isReversed`
- no if-else conditions which increase logic complexity
- yes, some code will be duplicated but it will look much clear and easier for understanding - just standalone dialog
- less error prone
- it's much easier to adjust separate dialogs rather than dialogs used in a lot of places, with complex conditions

Reply:
Adding new dialog for "End engagement" won't solve the whole issue atm. as same process needs to be done for "Exit queue". As we are changing only the part that is related to layout choice its safe to say its less error prone at the moment (as previous behaviour was tested and approved)

We plan refactoring of dialogs during multiple dialogs support task.